src/plugins/dashboard/public/application/dashboard_app.tsx

```diff
@@ -59,7 +59,7 @@ export function DashboardApp({
     indexPatterns: indexPatternService,
   } = useKibana<DashboardAppServices>().services;
 
-  const [lastReloadTime, setLastReloadTime] = useState(0);
+  const triggerRefresh$ = useMemo(() => new Subject<{ force?: boolean }>(), []);
```

Comment:
This is a nice improvement. Much better than using a `lastReloadTime` to trigger forced resets.